pkg/test/framework/features/features.yaml

```diff
@@ -56,5 +56,6 @@ features:
       plugin-cert:
   # features which allow extending or deep integrations with istio and other products
   extensibility:
+    vm:
```

Comment:
I don't think this is the right section, this is more for the telemetry/wasm stuff. I also don't know what the right section is @therealmitchconnors ?

Reply:
Will update once we decide what the best section is.